.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish Base Image
+
+on:
+  workflow_dispatch:
```

Comment:
comment: this will work I think but you can also do `on: [workflow_dispatch]` if there's no info needed under it.